components/prism-core.js

```diff
@@ -187,6 +248,30 @@ var _ = {
 		}
 	},
 
+	/**
+	 * Highlights the code inside a single element.
+	 *
+	 * The following hooks will be run:
+	 * 1. `before-sanity-check`
+	 * 2. `before-highlight`
+	 * 3. All hooks of {@link Prism.highlightElement}. These hooks will only be run by the current worker if `async` is `true`.
+	 * 4. `before-insert`
+	 * 5. `after-highlight`
+	 * 6. `complete`
+	 *
+	 * @param {HTMLElement} element The element containing the code.
```

Comment:
This doesn’t need to be an HTMLElement:
```suggestion
	 * @param {Element} element The element containing the code.
```